hedera-node/hedera-config/src/main/java/com/hedera/node/config/data/BlockStreamConfig.java

```diff
@@ -42,5 +42,6 @@ public record BlockStreamConfig(
         @ConfigProperty(defaultValue = "true") @NetworkProperty boolean compressFilesOnCreation,
         @ConfigProperty(defaultValue = "32") @NetworkProperty int hashCombineBatchSize,
         @ConfigProperty(defaultValue = "1") @NetworkProperty int roundsPerBlock,
+        @ConfigProperty(defaultValue = "2") @Min(0) @NetworkProperty int blockPeriodSeconds,
```

Comment:
Perhaps more elegant to use `Duration` as the config type since it is a unit of time?